### PR TITLE
Dashboard refresh at monitoring integration sync

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/monitoring-integration#260
tendrl-bugid: Tendrl/monitoring-integration#259

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
Tested

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n @rishubhjain @nthomas-redhat please review

File: tendrl/monitoring_integration/grafana/create_dashboards.py [hunk 1]
```diff
@@ -53,117 +60,30 @@ def get_cluster_details(integration_id=None):
...
-                    except (KeyError, etcd.EtcdKeyNotFound) as ex:
-                        logger.log("error", NS.get("publisher_id", None),
-                                   {'message': "Error while getting "
-                                    "volumelevel brick details" + str(ex)})
+            # Get node details
+            cluster_obj.hosts = get_node_details(cluster_key)
+            # Get volume details
+            cluster_obj.volumes = get_volumes_details(cluster_key)
+            # Get brick details from subvolumes
+            cluster_obj.bricks = get_brick_details(cluster_obj.volumes)
```
Comment (r0h4n) [inline tendrl/monitoring_integration/grafana/create_dashboards.py]: Nice work on simplifying this long method

Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/grafana/create_dashboards.py]: :+1:

**Pull request merged**